assets/gitlab/.gitlab-ci.yml

```diff
@@ -82,14 +60,13 @@ push-package-to-platform:
     name: torizonextras/xonsh-wrapper:dev
 
   variables:
-    TASKS_ITERATIVE: "False"
+    TASKS_DISABLE_INTERACTIVE_INPUT: "True"
     TASKS_OVERRIDE_ENV: "False"
 
   script: |
     echo $PLATFORM_CREDENTIALS | base64 -d > credentials.zip
     echo '
 
-      xonsh ./.vscode/tasks.xsh run run-torizon-binfmt
       xonsh ./.vscode/tasks.xsh run tcb-platform-publish
 
     ' | xonsh-wrapper.xsh
```

Comment:
I though that the idea was to use bash directly, no? So, I believe that here we do not need the `xonsh-wrapper` anymore.